src/components/UI/loading-spinner.tsx

```diff
@@ -1,23 +1,24 @@
-const LoadingSpinner = () => {
+import { faGear } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
```

Comment:
If possible, could you remove these packages and use the `react-icons` package already installed instead? It should be the exact same as https://react-icons.github.io/react-icons/search/#q=gear

Reply:
Done. Are we trying to remove fontawesome?